src/node_jsrtapi.cc

```diff
@@ -364,69 +403,78 @@ napi_status napi_set_property(napi_env e, napi_value o,
   return napi_ok;
 }
 
-napi_status napi_define_property(napi_env e, napi_value o,
-    napi_property_descriptor* p) {
-  napi_value descriptor;
-  CHECK_NAPI(napi_create_object(e, &descriptor));
-
+napi_status napi_define_properties(napi_env e,
+                                   napi_value o,
+                                   int property_count,
+                                   const napi_property_descriptor* properties) {
   napi_propertyname configurableProperty;
   CHECK_NAPI(napi_property_name(e, "configurable", &configurableProperty));
```

Comment:
Not related to your PR but I wonder if this should be napi_get_property_name, to be consistent with other NAPI APIs.

Also, more generally, I wonder what our philosophy should be with respect to NAPI implementations- this method for example generally calls other NAPI APIs- is there a reason for that vs. straight up calling JSRT APIs? In the end, I'm not sure it matters for the trivial NAPI wrappers since the optimizer should be able to inline those calls, but I was more curious if this decision was an intentional one or not.